src/NzbDrone.Core/Datastore/Migration/182_on_delete_notification.cs

```diff
@@ -0,0 +1,14 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(182)]
+    public class on_delete_notification : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Notifications").AddColumn("OnDelete").AsBoolean();
```

Comment:
Tests are failing due to this needing a default value.

Use `.WithDefaultValue()` method with a default in parenthesis